javascript-client/test/api/ProjectApi.spec.js

```diff
@@ -107,77 +72,65 @@
           ]
         };
 
-        var projectId;
+        instance.apiClient.authentications['Bearer'].apiKey = userToken;
+        instance.apiClient.authentications['Bearer'].apiKeyPrefix = "Bearer";
 
         instance.addProject(project, function (error, data, res) {
           if (error) throw error;
-          projectId = res.body.project._id;
-          instance.projectDetailsIdGet(projectId, function (error, res) {
+          let projectId = res.body.project._id;
+          instance.projectDetailsIdGet(projectId, function (error, data, res) {
+            if (error) throw error;
             expect(res.status).to.be(200);
             done();
           })
-          expect(res.status).to.be(200);
-          done();
         });
+      });
+      it('Admin CRUD successfully', async function (done) {
+        const userToken = await authUtil.loginUser('admin@admin.com', 'admin');
 
-        //uncomment below and update the code to test addProject
-        //instance.addProject(function(error) {
-        //  if (error) throw error;
-        //expect().to.be();
-        //});
-        //done();
+        let project = {
+          "title": "Admin Project",
+          "model": "Admin Model",
+          "year": 2010,
+          "description": "This is Admin test description",
+          "image": "string",
+          "comments": [
+            "string"
+          ]
+        };
+
+        let projectId;
+
+        instance.apiClient.authentications['Bearer'].apiKey = userToken;
```

Comment:
Avoid copy/paste :). If you have to copy/paste code that is a clear indication that you need to put this code in a function and you will be able to reuse it. It seems that you will have to do this operation frequently.